registry/alembic/versions/013efbf93251_org_publish_nonce.py

```diff
@@ -0,0 +1,28 @@
+"""org publish nonce
+
+Revision ID: 013efbf93251
+Revises: 59be957f84ad
+Create Date: 2020-05-28 21:35:10.452100
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '013efbf93251'
+down_revision = '59be957f84ad'
+branch_labels = None
+depends_on = None
+
+
```

Comment:
Please make sure that the DDL change is applied on all environments carefully

Reply:
@pratik-vii would you like to comment on this.

Reply:
It will be upgraded automatically in CD aws.